cookbooks/planet/files/default/cgi/HEADER.cgi

```diff
@@ -90,32 +89,42 @@ database, and those published before the 12 September 2012 are distributed under
         <p>%(planet_pbf_link)s</p>
         <p>
         Each week, a new and complete copy of all data in OpenStreetMap is made
-        available as both a compressed XML file and a custom PBF format file.
+        available as both a compressed XML file and a more compact
+        <a href="https://wiki.openstreetmap.org/wiki/PBF_Format">custom PBF format</a> file.
         Also available is the <a href="planet/full-history">'history'</a> file
         which contains not only up-to-date data but also older versions of data
         and deleted data items.
-        <p>
         </p>
+        <p>
         A smaller file with complete metadata for all changes ('changesets') in
         XML format is also available.
         </p>
     </td>
     <td>
         <p>
-        You are granted permission to use OpenStreetMap data by
-        <a href="https://osm.org/copyright">the OpenStreetMap License</a>, which also describes
-        your obligations.
+        This open data is <a href="https://osm.org/copyright">copyright &copy; OpenStreetMap</a>.
```

Comment:
I think this is the only substantive difference, as against rearranging or rewording of existing text.

I also think it's problematic because it's asserting that "OpenStreetMap" owns the copyright but there is no such entity and even the foundation is not the copyright holder (to the extent that copyright even matters as things like database rights are likely more important) as user's do not assign copyright in their contributions.

Reply:
The [recommended attribution text](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines#Attribution_text) now is just "OpenStreetMap", although it seems the "©" doesn't seem to be recommended now.

Maybe just "This open data is provided by OpenStreetMap", linking back to osm.org/copyright?

Reply:
Attribution is not the same as claiming to own the copyright of course.

Reply:
Yeah, unless I'm mistaken, the "copyright OpenStreetMap" wording was recommended for a while but it looks like that's changed.

I've tweaked the wording now.